test/specs/EventsSpec.js

```diff
@@ -224,6 +224,16 @@ describe("Event Tests", function() {
 
     });
 
+    describe("Window Events", function() {
+      it("'resize' event is triggered properly and can be binded to", function() {
+        testSlider.on('resize', function() {
+          flag = true;
+        });
+        testSlider.data('slider')._resize();
```

Comment:
Actually, try using `window.dispatchEvent( new Event('resize') )` in the test and see if that works. 

Since you are binding to the browser native event with `addEventListener()`, I am guessing that using the jQuery `trigger()` method only triggers jQuery-specific events.